syntax = "proto3";


message HelloQromaRequest {
  string name = 1;
}

message HelloQromaResponse {
  string response = 1;
  uint32 callCount = 2;
  uint32 nameLength = 3;
}